eng/patches/emscripten-version.txt

```diff
@@ -0,0 +1 @@
+"3.1.7"
```

Comment:
Can we generate this at build time? And drop the quotes

Reply:
This is copy of `emsdk/upstream/emscripten/emscripten-version.txt`. Maybe we can copy it from the docker image during the build.

Reply:
I copy it from `$(EMSDK_PATH)` now. The quotes are now stripped with sed on linux build https://github.com/dotnet/icu/pull/197/files#diff-3ef419c17802b9f6582ae68e20648c853b295a742f5437981b8f0b32f2b4aff2R35-R38. Do you know how to strip them on windows or with msbuild?

Reply:
I think an inline c# msbuild task might do it here. Will try it.

Reply:
Ah, the `CodeTaskFactory` is not available on dotnet msbuild. So used `ReadLinesFromFile`/`WriteLinesToFile` instead.